engine/src/flutter/shell/platform/darwin/macos/framework/Source/FlutterChannelKeyResponder.mm

```diff
@@ -104,9 +104,6 @@ - (void)syncModifiersIfNeeded:(NSEventModifierFlags)modifierFlags
             forEventFlags:modifierFlags
                   keyCode:0x00000039  // kVK_CapsLock
                 timestamp:timestamp];
```

Comment:
Why is this check removed?

Reply:
Because during CMD+TAB macOS for some reason sends event with `NSEventModifierFlagCommand` set, but without the left + right bits set. And because we only check for the left + right bits when synthesizing the event, in the end the comparison fails because the flags we built value doesn't correspond to the original one.